### DISCUSSION TITLE
Markdown Rendering Option

### DISCUSSION THREAD

**Discussion by hburrichter** [langfuse/langfuse#2097] [Ideas]
### Describe the feature or potential improvement

It would be great to see the rendered Markdown of a trace input/output or other observations in the Langfuse UI. 

This would greatly improve the readability of Markdown elements like tables and images. Currently, we need to copy and open image links manually to view and check them.

Perhaps a "Markdown" option could be added to the existing "Pretty | Json" button group in Langfuse?

**Comment by clemra** [comment]
That's a great suggestion @hburrichter -- we will look into it and see if we can make this work. Many people use markdown as output and obv. it would be easier to digest that way. 

Will update on this discussion.

**Comment by marcklingen** [comment]
love this, turned this into an issue to track this change: https://github.com/langfuse/langfuse/issues/2109

**Comment by chai3** [comment]
+1

@hburrichter @marliessophie @marcklingen 
For my use case it would be nice if the dataset item view's input and expected output could also be rendered markdown.
But I don't know if my use case is popular.

**Reply by marcklingen** [reply]
we'll add markdown support across tracing soon, extending this to datasets should be simple then

**Comment by marcklingen** [comment]
Shipped in July: https://langfuse.com/changelog/2024-07-15-render-markdown-trace-preview
